docs/source/methods/avas.rst

```diff
@@ -168,6 +168,241 @@ We then use this guess of active orbitals to compute the CASCI energy:::
+because neither of the three directions is perpendicular to the π plane.
+This leads to the following input section of AVAS: ::
+
+  set forte {
+    subspace           ["C(2p)","O(2p)"]  # must include all 2p orbitals!
+    subspace_pi_planes [["C","O","H"]]    # only one plane, defined by all C, O and H atoms
+    avas               true
+    avas_diagonalize   true
+    avas_sigma         1.0
+  }
+
+and the output is now identical to the very first example ::
+
+  ==> Atomic Valence MOs (Active Marked by *) <==
+
+    ===============================
+     Irrep    MO  Occ.  <phi|P|phi>
+    -------------------------------
+    *   A      0    2      0.970513
+    *   A      8    0      0.992548
+    *   A      9    0      0.022209
+    ===============================
+
+Some comments on the expressions of :code:`SUBSPACE_PI_PLANES` are necessary.
+Some valid expressions to define the π planes would be: ::
+
+  - [['C', 'H', 'O']]              # only one plane consisting all C, H, and O atoms of the molecule.
+  - [['C1-6'], ['N1-2', 'C9-11']]  # plane 1 with the first six C atoms of the molecule,
+                                   # plane 2 with C atoms #9, #10, and #11, and N atoms #1 and #2.
+  - [['C1-4'], ['C1-2', 'C5-6']]   # plane 1 with the first four C atoms of the molecule,
+                                   # plane 2 with C atoms #1, #2, #5, and #6.
+                                   # Two planes share C1 and C2!
+
+This syntax follows the one used by :code:`SUBSPACE`: ::
+
+  - ["C"]              # all carbon atoms
+  - ["C","N"]          # all carbon and nitrogen atoms
+  - ["C1"]             # carbon atom #1
+  - ["C1-3"]           # carbon atoms #1, #2, #3
+  - ["C(2p)"]          # the 2p subset of all carbon atoms
+  - ["C(1s)","C(2s)"]  # the 1s/2s subsets of all carbon atoms
+  - ["C1-3(2s)"]       # the 2s subsets of carbon atoms #1, #2, #3
+  - ["Ce(4fzx2-zy2)"]  # the 4f zxx-zyy orbital of all Ce atoms
+
+.. note::
+    It is very important to include a complete set of p orbitals in :code:`SUBSPACE` if π planes are defined.
+    Otherwise, the code will follow the directions given by :code:`SUBSPACE`.
+
+.. tip::
+    The code is flexible enough to treat double active spaces (e.g., double π or double d-shell).
+    For example, the double-π active space of formaldehyde can be obtained via ::
+
+      set forte {
+        minao_basis        double-shell
+        subspace           ["C(2p)","C(3p)","O(2p)","O(3p)"]
+        subspace_pi_planes [["C","O","H"]]
+        avas               true
+        avas_diagonalize   true
+        avas_cutoff        0.5
+      }
+
+    Here I prepare a basis called "double-shell.gbs", which includes the 2p and 3p orbitals of C and O atoms.
+    You can also prepare your own MINAO basis by truncating the the cc-pVTZ or ANO-RCC-VTZP basis sets.
+
+For a more realistic example, consider the following iron porphyrin related molecule:
```

Comment:
I think this should be a new section specifically for molecules with two or more pi planes. You also need to explain (perhaps at the end, how AVAS uses this information). There are a lot of technical details at the beginning of the text, but here we do not explain what the multiple planes do or why they are necessary.

Reply:
I added a new section. Hope the change matches your standard.